liquid_tags/requirements.txt

```diff
@@ -0,0 +1 @@
+pytest
```

Comment:
Can `nose` run `py.test` tests?

Reply:
Not the ones defined as simple functions no, but the standard ones defined as classes yes: https://github.com/getpelican/pelican-plugins/blob/master/liquid_tags/test_generation.py

At some point it may be interesting to switch from `nose` to `pytest` as the former one is not much maintained anymore, but it should probably be done in another PR

Reply:
Thanks :+1: